rented-properties/src/main/java/org/egov/cpt/util/PropertyUtil.java

```diff
@@ -199,4 +203,14 @@ public String getTransitNumberFromConsumerCode(String consumerCode) {
 		}
 		return null;
 	}
+
+	@SuppressWarnings("unchecked")
+	public long getInterstStartFromMDMS(Property property, RequestInfo requestInfo) {
+		String filter = "[?(@.code=='" + property.getColony() +"')].interestStartYear"; 
+		Map<String, List<Long>> interestStartDateList =(Map<String, List<Long>>)mdmsService.getMDMSResponse(requestInfo,
```

Comment:
Calls to mdms doesn't require a valid user or access token. So we can construct a dummy requestInfo object to send to it without getting that as argument.